Themes/default/languages/Post.english.php

```diff
@@ -64,7 +64,7 @@
 // Use numeric entities in the below two strings.
 $txt['reported_post'] = 'Reported post';
 $txt['reported_to_mod_by'] = 'by';
-$txt['rtm10'] = 'Submit';
+$txt['submit_report'] = 'Submit';
```

Comment:
It would be more consistent with our naming conventions if this were `$txt['report_submit']` instead of `$txt['submit_report']`